example.yaml

```diff
@@ -221,6 +221,13 @@ kafka_metric_topic: ""
 # Name of the topic we'll be publishing spans to
 kafka_span_topic: "veneur_spans"
 
+# Name of a tag to hash on for sampling; if empty, all spans are sampled
+kafka_span_sample_tag: ""
+
+# Set to the floating point sampling rate on a particular tag.
+# If set above 1.0, this will default to 1.0
+kafka_span_sample_rate: 1.1
```

Comment:
Is this 1.1 so that it gets set to the correct type from gojson?

Reply:
Yep. I opened https://github.com/ChimeraCoder/gojson/pull/59 and will change this to 1.0 once that’s done.